### PR TITLE
7pm 2 update admin/teams placeholder

### PR DESCRIPTION
**Pull request by kaiwenli1**
Replace the placeholder in /admin/teams with teamname, id and description. Showing list of all the teams that are in database.
image

### CODE REVIEW

**Review by JohnnJudge — APPROVED** [review]
Everything looks good to me.

File: javascript/src/main/pages/Admin/Teams.js [hunk 1]
```diff
@@ -1,46 +1,29 @@
...
-                    <td>PlaceHolder1</td>
-                    <td>TeamPlaceHolder</td>
-                </tr>
-                <tr>
-                    <td>PlaceHolder2</td>
-                    <td>TeamPlaceHolder</td>
-                </tr>
-            </tbody>
-        </Table>
+        <BootstrapTable keyField='teamName' data={teamNames || []} columns={columns} />
```
Comment (thiskappaisgrey) [inline javascript/src/main/pages/Admin/Teams.js:26]: There seems to be no tests associated with these lines of code. The tests present just makes sure it renders, which is insufficient. I think for this to be merged, there should be better tests written, but don't worry about it too much.

**Review by thiskappaisgrey — CHANGES_REQUESTED** [review]
You should decide if you want to write tests for the frontend part. I think you should, but if you don't have time, you can ignore this review.

File: javascript/src/main/pages/Admin/Teams.js (hunk 1 above)
Comment (kaiwenli1) [reply javascript/src/main/pages/Admin/Teams.js:26]: I checked the slackUsers.test.js doesn't contain any testcase either. So I just leave the test case like that.

Comment (thiskappaisgrey) [reply javascript/src/main/pages/Admin/Teams.js:26]: ok that's fine I guess.

**Review by jayleenli — APPROVED** [review]

**Pull request merged**